test/test_spectral_ops.py

```diff
@@ -0,0 +1,584 @@
+import torch
+import unittest
+import math
+from contextlib import contextmanager
+from itertools import product
+
+from torch.testing._internal.common_utils import \
+    (TestCase, run_tests, TEST_NUMPY, TEST_LIBROSA)
+from torch.testing._internal.common_device_type import \
+    (instantiate_device_type_tests, dtypes, onlyOnCPUAndCUDA, precisionOverride,
+     skipCPUIfNoMkl, skipCUDAIfRocm, deviceCountAtLeast, onlyCUDA)
+
+if TEST_NUMPY:
+    import numpy as np
+
+
+if TEST_LIBROSA:
+    import librosa
+
+# saves the torch.fft function that's clobbered by importing the torch.fft module
+fft_fn = torch.fft
+import torch.fft
+
+# Tests of functions related to Fourier analysis in the torch.fft namespace
+class TestFFT(TestCase):
+    exact_dtype = True
+
+    def test_fft_function_clobbered(self, device):
+        t = torch.randn((100, 2), device=device)
+        eager_result = fft_fn(t, 1)
+
+        def method_fn(t):
+            return t.fft(1)
+        scripted_method_fn = torch.jit.script(method_fn)
+
+        self.assertEqual(scripted_method_fn(t), eager_result)
+
+        with self.assertRaisesRegex(TypeError, "'module' object is not callable"):
+            torch.fft(t, 1)
+
+    @unittest.skipIf(not TEST_NUMPY, 'NumPy not found')
```

Comment:
it looks like you are moving and changing in one PR (i.e. I see torch.fft.fft which doesn't exist currently).  I don't think we should be doing that -- it seems totally fair to move everything in a previous PR.

Reply:
This was addressed with https://github.com/pytorch/pytorch/pull/42157.